src/fides/data/privacy_notices/privacy_notice_templates.yml

```diff
@@ -137,6 +137,7 @@ privacy_notices:
     - is
     - "no"
     - li
+    - ca_qc
```

Comment:
We have intentional behavior to not update existing notices automatically.   If we do in fact end up adding ca_qc here, I should add a migration for you, let me know the outcome here ^

Current behavior explanation:

On startup, we create or update our "templates" in the db from the notices in this yaml file, but then only create new notices from new templates, we don't update existing notices.  This is because customers could have edited their notices, and a restart would wipe their changes and reset back to the original state.

Reply:
More of a note to self: if doing a data migration, we need to make sure all the same things happen when you create privacy notices out of the box.  Main thing is that you need to make sure a ca_qc experience is created for each component type.  I think here, just the notice itself needs to be updated, because the proper experiences are created with the other notices below.   Double check if there's anything I'm missing here.

Reply:
@NevilleS , I talked to @mfbrown  today about this one. We had an Essential notice and we had a Functional notice. This one combines the two so the determination was to have a different notice.
The requirements for the Analytics notice were the exact same so it was easy (from PM perspective) to add the geography. Make sense?

Reply:
@pattisdr, regardless of what comes from the conversation with @NevilleS , we did add `ca_qc` to the Analytics notice. Is that a problem?

Reply:
https://github.com/ethyca/fides/pull/4004#discussion_r1311954849

Updating an existing template here does not automatically make updates for existing customers.